src/backend_skia/image.rs

```diff
@@ -49,7 +49,7 @@ pub fn draw_raster(
     };
 
 
-    let mut filter = skia::FilterQuality::Low;
+    let mut filter = skia::FilterQuality::Medium;
```

Comment:
This is a fix for embedded raster quality that was previously here: https://github.com/velyan/resvg/pull/9